Cargo test results in compile error on Windows

```
$ pwd
/g/rust/wgpu

$ cargo test
   Compiling wgpu-native v0.2.0 (G:\rust\wgpu\wgpu-native)
error[E0425]: cannot find value `raw` in this scope
   --> wgpu-native\src\instance.rs:122:24
    |
122 |     SurfaceHandle::new(raw)
    |                        ^^^ not found in this scope
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,6 +9,9 @@ matrix:
     - os: linux
       rust: nightly
       compiler: gcc
+    # Windows 64bit
+    - os: windows
+      rust: stable
 
     # macOS 64bit
     - env: MACOSX_DEPLOYMENT_TARGET=10.9
@@ -28,10 +31,6 @@ matrix:
     #  osx_image: xcode9
     #  rust: nightly
 
-    # Windows 64bit
-    #- os: windows
-    #  rust: stable
-
 branches:
   except:
   - staging.tmp
@@ -40,8 +39,8 @@ before_install:
   # Do not run bors builds against the nightly compiler.
   # We want to find out about nightly bugs, so they're done in master, but we don't block on them.
   - if [[ $TRAVIS_RUST_VERSION == "nightly" && $TRAVIS_BRANCH == "staging" ]]; then exit; fi
+  - if [[ $TRAVIS_OS_NAME == "windows" ]]; then choco install make; fi
 
 script:
   - cargo test
-  - cargo check --manifest-path wgpu-native/Cargo.toml
   - if [[ $TRAVIS_OS_NAME == "osx" ]]; then (brew update && brew upgrade cmake && brew install glfw3 && cd wgpu-native && cargo build --features=local,gfx-backend-metal && cd ../examples/hello_triangle_c && mkdir build && cd build && cmake .. && make); fi

--- a/wgpu-native/src/instance.rs
+++ b/wgpu-native/src/instance.rs
@@ -109,7 +109,11 @@ pub fn instance_create_surface_from_windows_hwnd(
     hinstance: *mut std::ffi::c_void,
     hwnd: *mut std::ffi::c_void,
 ) -> SurfaceHandle {
-    #[cfg(not(target_os = "windows"))]
+    #[cfg(not(any(
+        feature = "gfx-backend-dx11",
+        feature = "gfx-backend-dx12",
+        all(target_os = "windows", feature = "gfx-backend-vulkan"),
+    )))]
     let raw = unimplemented!();
 
     #[cfg(any(feature = "gfx-backend-dx11", feature = "gfx-backend-dx12"))]
@@ -118,7 +122,7 @@ pub fn instance_create_surface_from_windows_hwnd(
     #[cfg(all(target_os = "windows", feature = "gfx-backend-vulkan"))]
     let raw = HUB.instances.read()[instance_id].create_surface_from_hwnd(hinstance, hwnd);
 
-    #[cfg_attr(not(target_os = "windows"), allow(unreachable_code))]
+    #[allow(unreachable_code)]
     SurfaceHandle::new(raw)
 }
 

